.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
```

Comment:
ah interesting. Thats really confusing though if you don't know how old your repo is...